package.json

```
@@ -41,7 +41,9 @@
             "default": "./out/web.mjs"
         },
         "./types": {
-            "types": "./out/types.d.ts"
+            "types": "./out/types.d.ts",
+            "node": "./out/types.node.js",
+            "default": "./out/types.js"
```

In the npm package, `types.js` simply re-exports everything from `types.node.js`. Can we perform the following change and still have it work?
```suggestion
            "node": "./out/types.js",
            "default": "./out/types.js"
```

---

And if yes, then it should also be possible to drop the node-specific export entirely, right?

---

In fact, @kiki-kanri suggested the same thing in https://github.com/grammyjs/hydrate/issues/24#issuecomment-1873433145

---

```suggestion
            "default": "./out/types.js"
```